daemon/meson.build

```diff
@@ -1,105 +1,80 @@
-# Machine Learing Agent
-if get_option('enable-ml-service')
-  nns_ml_agent_incs = include_directories('includes')
-  nns_ml_agent_srcs = files('main.c', 'modules.c', 'gdbus-util.c',
-    'service-db.cc', 'pipeline-module.cc', 'model-dbus-impl.cc')
-  if get_option('enable-tizen')
-    nns_ml_agent_srcs += files('pkg-mgr.c')
-  endif
-
-  # Generate GDbus header and code
-  gdbus_prog = find_program('gdbus-codegen', required : true)
-  gdbus_gen_pipeline_src = custom_target('gdbus-gencode',
-    input : pipeline_dbus_input,
-    output : ['pipeline-dbus.h', 'pipeline-dbus.c'],
-    command : [gdbus_prog, '--interface-prefix', 'org.tizen',
-              '--generate-c-code', 'pipeline-dbus',
-              '--output-directory', meson.current_build_dir(),
-              '@INPUT@'])
-
-  gdbus_gen_model_src = custom_target('gdbus-model-gencode',
-    input : model_dbus_input,
-    output : ['model-dbus.h', 'model-dbus.c'],
-    command : [gdbus_prog, '--interface-prefix', 'org.tizen',
-              '--generate-c-code', 'model-dbus',
-              '--output-directory', meson.current_build_dir(),
-              '@INPUT@'])
-
-  gdbus_gen_header_dep = declare_dependency(sources: [gdbus_gen_pipeline_src, gdbus_gen_model_src])
+# Machine Learning Agent
+# This file should be included when ml-service feature is enabled.
+nns_ml_agent_incs = include_directories('includes')
+nns_ml_agent_srcs = files('main.c', 'modules.c', 'gdbus-util.c',
+  'service-db.cc', 'pipeline-dbus-impl.cc', 'model-dbus-impl.cc')
+if get_option('enable-tizen')
+  nns_ml_agent_srcs += files('pkg-mgr.c')
+endif
 
-  # Enable ML Agent Test
-  if get_option('enable-test')
-    gdbus_gen_test_src = custom_target('gdbus-gencode-test',
-      input : test_dbus_input,
-      output : ['test-dbus.h', 'test-dbus.c'],
-      command : [gdbus_prog, '--interface-prefix', 'org.tizen',
-                '--generate-c-code', 'test-dbus',
-                '--output-directory', meson.current_build_dir(),
-                '@INPUT@'])
+# Generate GDbus header and code
+gdbus_prog = find_program('gdbus-codegen', required : true)
```

Comment:
If require is true meson will abort if no program can be found.
I understand that this was intended.

Reply:
Right, ml-service daemon requires this.